dev/com.ibm.ws.threading_policy_fat/test-applications/vtpolicyapp/src/web/vt/PolicyVirtualThreadServlet.java

```diff
@@ -319,4 +329,39 @@ public void testMaxPolicyStrictWithVirtualThreads() throws Exception {
 
         executor.shutdownNow();
     }
+
+    /**
+     * Tests a vt override is implemented via a userFeature (or any stackproduct feature)
+     *
+     */
+    public void testDisableVirtualThreads() throws Exception {
+        boolean thrown = false;
+
+        Map<String, Object> config = new TreeMap<>();
+        config.put("max", 3);
+        config.put("maxPolicy", MaxPolicy.strict.name());
+        config.put("virtual", true);
+        // defaults:
+        config.put("expedite", 0);
+        config.put("maxWaitForEnqueue", 0L);
+        config.put("runIfQueueFull", false);
+
+        PolicyExecutor executor = provider.create("testDisableVirtualThreads");
+        //First test is to ensure setting to virtual will throw exception
+        try {
+            executor.updateConfig(config);
+        } catch (IllegalArgumentException e) {
+            thrown = true;
+        }
+        assertTrue(thrown);
```

Comment:
I thought the point of the override was to force the use of platform threads everywhere. By rejecting virtual=true on the policy executor instead of rejecting it, this forces it back onto every component using the policy executor (and maybe some of them aren't even in our own product) to become aware of the override itself and ensure it sends through a `false` value to ensure the override is honored.  What are the reasons behind doing it this way instead?

Reply:
latest commit resolves this (removes assert for exception)